recipes/HydroErr/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "HydroErr" %}
+{% set version = "1.22" %}
+{% set sha256 = "c2521e960c82c4a2ff5f8e48155be785d817c861b97241135cbb524b74366600" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
This can be a `noarch` package. Please add `noarch: python`; see https://conda-forge.org/docs/meta.html#building-noarch-packages